core/src/main/java/io/confluent/rest/RestConfig.java

```diff
@@ -481,6 +481,15 @@ public class RestConfig extends AbstractConfig {
           + "Default is false.";
   protected static final boolean PROXY_PROTOCOL_ENABLED_DEFAULT = false;
 
+  public static final String ERROR_HANDLING_ENABLED_CONFIG = "error.handling.enabled";
+
+  protected static final String ERROR_HANDLING_ENABLED_DOC =
+      "If true, enable overall error handling for any uncaught errors in handlers pipeline. "
+          + "Exceptions usually display stack trace to the client and the error handler will "
+          + "disable the stack trace to the client.";
```

Comment:
If true, enable overall error handling for any uncaught errors in handlers pipeline. 

This ensures that no stack traces are included in responses sent to clients.